subsys/bluetooth/host/classic/a2dp.c

```diff
@@ -195,12 +195,14 @@ static int a2dp_set_config_ind(struct bt_avdtp *session, struct bt_avdtp_sep *se
 	int err;
 
 	*errcode = 0;
-	ep = CONTAINER_OF(sep, struct bt_a2dp_ep, sep);
-	if (ep == NULL) {
+
+	if (sep == NULL) {
```

Comment:
Wouldn't an assert be more appriate? It looks like avctp.c is the caller of this function, and it already ensures that it's never called with NULL sep.

Reply:
Yes. The `sep` should never be `NULL` here.

Reply:
Updated